Find a functionally equivalent code:com.android.layoutlib.bridge.impl.RenderSessionImpl.removeChild:COMMENT
Method Modifier: public      
Comment:/**
 * Removes a child from its current parent.
 * <p>
 * {@link #acquire(long)} must have been called before this.
 *
 * @throws IllegalStateException if the current context is different than the one owned by
 * the scene, or if {@link #acquire(long)} was not called.
 *
 * @see RenderSession#removeChild(Object, IAnimationListener)
 */

@@ -20,26 +20,26 @@
         return SUCCESS.createResult();
     }
     Result result = removeView(parent, childView);
-    if (result.isSuccess() == false) {
+    if (!result.isSuccess()) {
         return result;
     }
     return render(false);

